report/db.py

```diff
@@ -4,6 +4,6 @@
 SESSION = sessionmaker()
 
 
-def make_engine(settings):
+def make_engine(settings):  # pragma: no cover
```

Comment:
We only use this in the functest.

We could adapt it there but I reckon we'd bring it back at some point and forget about the other one.